MmSupervisorPkg/Core/MmSupervisorCore.inf

```diff
@@ -234,7 +234,7 @@
 [BuildOptions.common]
   #Subsystem version will be used as MmSupervisor driver version
   #Image version will be used as MmSupervisor security patch level(SPL) value
-  MSFT:RELEASE_*_*_DLINK_FLAGS  = /DLL /SUBSYSTEM:CONSOLE,$(RELEASE_DRIVER_VERSION) /VERSION:$(MAJOR_PATCH_LEVEL).$(MINOR_PATCH_LEVEL)
+  MSFT:RELEASE_*_*_DLINK_FLAGS  = /DLL /SUBSYSTEM:CONSOLE,$(RELEASE_DRIVER_VERSION) /VERSION:$(MAJOR_PATCH_LEVEL).$(MINOR_PATCH_LEVEL) /MERGE:.bss=.none

```

Comment:
With this, do you see the .bss section called ".none" in the binary?

Reply:
In the map file it still appears as .bss and class type DATA.